scripts/build_and_test.sh

```diff
@@ -15,9 +15,13 @@ BUILD_AND_TEST_TEST_SUITE_ID=771f3f55-7db9-11e5-80ff-22000b5181e9
 start_test_suite ${BUILD_AND_TEST_TEST_SUITE_ID}
  
 function build_and_test() {
-  ${MAVEN} clean deploy -s "${MAVEN_SETTINGS}"
+    npm config set strict-ssl false
+    npm install
+    npm run start:mock-okta
+    ${MAVEN} clean package -s "${MAVEN_SETTINGS}"
```

Comment:
But CI will be running this script right ? which will need to sign/deploy the build.

Locally you should only need to `install` at most.  I'm not sure how CI is setup, but we may need one script for CI and one for dev (the or a flag in the build_and_test.sh script to not deploy)